sitemedia/js/components/RangeFacet.js

```diff
@@ -0,0 +1,38 @@
+import { mapActions } from 'vuex'
+
+export default Vue.component('RangeFacet', {
+    template: `
+    <div class="range-facet">
+        <div class="inputs">
+            <sui-input :placeholder="minVal" />
```

Comment:
`FilterChoice` actually isn't used currently since filters aren't implemented. I went back and forth between having a component to represent a single choice/checkbox (essentially a custom wrapper around `<input>`) vs just using `<sui-input>`...most of that struggle was prior/during/the cause of me implementing Vuex, because having the checkbox maintain its own state and simultaneously publish that state to the form wasn't working well.

The reason using semantic-ui vue is necessary at all is because the behaviors that semantic UI provides (like the js used for the interactivity on the site search bar and mobile menu) might not play well if used inside a Vue component.